# Generated from: /home/atty/Prog/__IGN__/2015_LI3DS/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogMag.h
# SbgLogMagCalib
uint32 timeStamp
uint16 reserved
uint8[16] magData